vnext/Microsoft.ReactNative/Views/VirtualTextViewManager.cpp

```diff
@@ -96,6 +96,11 @@ bool VirtualTextViewManager::UpdateProperty(
       NotifyAncestorsTextPropertyChanged(node, PropertyChangeType::AddPressable);
     }
   } else {
+    if (propertyName == "onMouseEnter" || propertyName == "onMouseLeave" && !propertyValue.IsNull()) {
```

Comment:
This should probably be: `propertyValue.AsBoolean()` instead of `!propertyValue.IsNull()`.